.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: All the CI
+
+on: [push, pull_request]
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x]
+    steps:
```

Comment:
Thinking of dropping this. Github has 10.x already installed, so I don't think we need to worry about specifying a node version. This step took a long time in the actions run